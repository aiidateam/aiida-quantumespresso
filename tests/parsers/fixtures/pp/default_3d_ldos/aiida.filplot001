 Energy =   6.4604 eV, broadening =   0.0100 eV
      36      36      36      36      36      36       2       1
     0        7.25711524      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
   0.0000000000000000       0.70710678118654757       0.70710678118654757
  0.70710678118654757        0.0000000000000000       0.70710678118654757
  0.70710678118654757       0.70710678118654757        0.0000000000000000
      320.1691947183        8.0000000000       30.0000000000     3
   1   Si    4.00
   1       0.707106781    0.000000000    0.000000000    1
   2       0.353553391    0.353553391    1.060660172    1
  4.146747864E-83  4.138790916E-83  4.117929905E-83  4.090864750E-83  4.065848830E-83
