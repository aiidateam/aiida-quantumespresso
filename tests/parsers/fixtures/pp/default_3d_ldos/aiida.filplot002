 Energy =   6.5604 eV, broadening =   0.0100 eV
      36      36      36      36      36      36       2       1
     0        7.25711524      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
   0.0000000000000000       0.70710678118654757       0.70710678118654757
  0.70710678118654757        0.0000000000000000       0.70710678118654757
  0.70710678118654757       0.70710678118654757        0.0000000000000000
      320.1691947183        8.0000000000       30.0000000000     3
   1   Si    4.00
   1       0.707106781    0.000000000    0.000000000    1
   2       0.353553391    0.353553391    1.060660172    1
  7.099397548E-31  7.081864325E-31  7.033817604E-31  6.964616300E-31  6.883652142E-31
