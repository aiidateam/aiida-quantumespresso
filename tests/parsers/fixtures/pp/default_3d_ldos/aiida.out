
     Program POST-PROC v.7.2 starts on 31Mar2025 at  7:48:51 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       1 processor cores
     Number of MPI processes:                 1
     Threads/MPI process:                     1

     MPI processes distributed on     1 nodes
     5407 MiB available memory on the printing compute node when the environment starts


     Reading xml data from directory:

     ./out/aiida.save/

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBESOL
                           (   1   4  10   8   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum         859     433    139                16889     5985    1067

     Using Slab Decomposition


     Calling punch_plot, plot_num =   3

     Energy =   6.46045 eV, broadening =   0.01000eV
     Writing data to file  aiida.filplot001

     Calling punch_plot, plot_num =   3

     Energy =   6.56045 eV, broadening =   0.01000eV
     Writing data to file  aiida.filplot002
     Reading data from file  aiida.filplot001

     Writing data to be plotted to file aiida.filplot001aiida.fileout
     Plot Type: 3D                     Output format: Gaussian cube
     Reading data from file  aiida.filplot002

     Writing data to be plotted to file aiida.filplot002aiida.fileout
     Plot Type: 3D                     Output format: Gaussian cube

     POST-PROC    :      0.46s CPU      0.49s WALL


   This run was terminated on:   7:48:51  31Mar2025            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
